### PR TITLE
Feat/card component

### PR DESCRIPTION
**Pull request by ocsiddisco**

### CODE REVIEW

**Review by Alexandra2888 — APPROVED** [review]

File: src/components/recipeItem/RecipeItem.module.css
Comment (NishaVijai) [inline src/components/recipeItem/RecipeItem.module.css:1]: I have a doubt. Can we write @media query as it is written here? I remember this format is written in .scss file, not in .css. Maybe I am wrong :-) Is this @media query showing the correct result as expected on the browser?

Comment (ocsiddisco) [reply src/components/recipeItem/RecipeItem.module.css:1]: It does work as expected yes.

Comment (NishaVijai) [reply src/components/recipeItem/RecipeItem.module.css:1]: Perfect :-)

**Review by NishaVijai — APPROVED** [review]
Thank you for the changes :-) Can be merged.

**Pull request merged**